content/fr/ingress.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Ingress
+status: Completed
+category: technologie
+tags: ["fondamental"]
+---
+
+Un Ingress est un ensemble de règles qui participe à gérer le trafic Internet depuis l'extérieur vers un conteneur ou un groupe de conteneurs s'exécutant dans un cluster.
+Il se compose de deux éléments : la ressource Ingress et le contrôleur Ingress. 
+La ressource Ingress est un fichier de configuration qui coexiste avec les autres fichiers manifestes et permet aux administrateurs de configurer le routage du trafic externe. 
+Le contrôleur Ingress est un serveur web qui met en place le routage du trafic selon la configuration de la ressource Ingress.
+
+## Problème auquel il répond
+
+Les applications web Cloud Native sont composées de plusieurs services, et souvent, ces [services](/fr/service/) doivent être accessibles depuis Internet pour que les utilisateurs puissent les visiter depuis leur navigateur. 
+Pour rendre ces services accessibles aux utilisateurs tout en utilisant [Kubernetes](/fr/kubernetes/) pour exécuter cette application, nous devons exposer chaque service applicatif au monde extérieur. 
+La manière la plus directe serait d'utiliser un Service d'équilibrage de charge Kubernetes (Load Balancer). 
+La création d'un tel service d'équilibrage de charge Kubernetes ajoute un nouveau composant dans l'infrastructure sous-jacente. 
```

Comment:
ici c’est dans la VO qu'il fraudais supprimer la répétition ^^

Reply:
Je vais faire une demande de PR sur la VO.

Reply:
Wait and See : https://github.com/cncf/glossary/issues/2880